Automattic/wp-calypso
# Domains: Add link to manage all domains

#### Changes proposed in this Pull Request

This PR introduces a link to manage all domains of the user.

<img width="1046" alt="Screenshot 2020-06-03 at 6 29 42 PM" src="https://user-images.githubusercontent.com/13062352/83662896-6ed95300-a5c8-11ea-99f7-ebea8da46656.png">


#### Testing instructions

Note: You might need to enable the flag by appending `?flags=manage/all-domains` to the URL.

- Apply the patch
- Switch to a site of your choosing, and go to Manage -> Domains.
- You should see the `Manage all your domains` button. Clicking it should send you to the all domains view
- Users with a single site should not see this link
- This link should not appear when in the all domains view
